### PR TITLE
FDS Example and Docs

### PR DESCRIPTION
**Pull request by wk9874**
- Created new Docker container
- Added entry to Integrations page for FDS connector
- Added FDS example

@kzscisoft could you remove your fds-example docker container please?

### CODE REVIEW

**Comment by AbyAbraham21** [comment]
Apologies its taken so long to revert back on this. Running a few examples using these docs so can comment once done

File: docs/examples/fds.md [hunk 1]
```diff
@@ -0,0 +1,342 @@
...
+            fds_input_file_path = "/workdir/input_no_vents.fds",
+            workdir_path = f"results_no_vents"
+        )
+
+    ```
+
+### Running the Simulation
+To run our initial simulation of the fire in the room without any vents, run the following command in the docker container:
+```
+python run_simulation_no_vents.py
```
Comment (AbyAbraham21) [inline docs/examples/fds.md:200]: @wk9874 : Will be worth adding a warning to have a user token or point to a reference to the same else the user will get an exception

**Review by AbyAbraham21 — CHANGES_REQUESTED** [review]
A couple of minor changes and then it is good to go

File: docs/examples/fds.md (hunk 1 above)
Comment (wk9874) [reply docs/examples/fds.md:200]: This is already in the setup instructions above:

```
### Update Simvue Config File
Finally we need to update the config file inside the Docker container to use your credentials. Login to the web UI, go to the **Runs** page and click **Create new run**. You should then see the credentials which you need to enter into the `simvue.ini` file. Simply open the existing file using `nano simvue.ini`, and replace the contents with the information from the web UI.
```

**Pull request merged**